expenses/models.py

```diff
@@ -23,21 +23,21 @@ class Category(models.TextChoices):
     description = models.TextField(max_length=250, default='')
     category = models.CharField(max_length=32, choices=Category.choices, default=Category.OTHER)
     house_name = models.ForeignKey(House, on_delete=models.CASCADE)
-    amount = models.IntegerField(null=True, blank=True, validators=[MinValueValidator(0)])
+    amount = models.IntegerField(null=False, blank=False, validators=[MinValueValidator(0)])
     date = models.DateTimeField(default=timezone.now)
 
     def __str__(self):
         return f'Category:{self.category},Amount:{self.amount}'
 
     @staticmethod
-    def create_expense(house_name, amount, date, category):
-        expense = Expenses(house_name=house_name, amount=amount, date=date, category=category)
+    def create_expense(house_name, amount, date, category, description):
```

Comment:
Can you do that the description has a default value of an empty string? 
```suggestion
    def create_expense(house_name, amount, date, category, description=''):
```

Reply:
yes, fixed